### PR TITLE
Fix chntpw compile on M1

### PR DESCRIPTION
**Pull request by itsmattkc**
I'm not 100% sure if this is the best solution, but it does work. Homebrew on M1 appears to use the prefix `/opt/homebrew` rather than `/usr/local/opt` used previously and hardcoded into the chntpw Makefile, leading to the following error:

```
clang: error: no such file or directory: '/usr/local/opt/openssl/lib/libcrypto.a'
```

This fix checks if the CPU is ARM, and forces the OpenSSL path to `/opt/homebrew` in chntpw if so. There might be a cleaner solution by checking the Homebrew version or something like that, but I'm not overly familiar with how that might work.

### CODE REVIEW

**Comment by mastarstrange** [comment]
Please fix this.
🤕

File: Formula/chntpw.rb [hunk 1]
```diff
@@ -8,7 +8,12 @@ class Chntpw < Formula
   depends_on "sidneys/homebrew/openssl@1.0"
 
   def install
-    system "make"
+    if Hardware::CPU.arm?
+      # Homebrew on M1 appears to install here instead of /usr/local/opt
+      system "make", "OSSLPATH=/opt/homebrew/opt/openssl@1.0"
```
[the remark this answers is not in the document]
Comment (itsmattkc) [reply Formula/chntpw.rb]: Done

**Comment by ntindle** [comment]
Is there a reason this can't be merged?

**Comment by mariadb-JuanVera** [comment]
All I did was to find the openssl1.0 version of the lib, symlink the old path to it, and the run the install, and it completed fine:

```
sudo find /opt -name "libcrypto.a"
    /opt/homebrew/Cellar/openssl@1.1/1.1.1m/lib/libcrypto.a
    /opt/homebrew/Cellar/openssl@1.0/1.0.2u/lib/libcrypto.a
sudo mkdir /usr/local/opt
sudo ln -s /opt/homebrew/Cellar/openssl@1.0/1.0.2u /usr/local/opt/openssl
brew install sidneys/homebrew/chntpw
sudo rm -fr /usr/local/opt
```

**Pull request merged**